packages/shared/src/components/modals/NewSquadModal.tsx

```diff
@@ -77,6 +84,42 @@ function NewSquadModal({
     if (shouldQuit) onRequestClose();
     return null;
   };
+
+  if (shouldShowBetaModal) {
+    return (
+      <Modal
+        isOpen
+        kind={Modal.Kind.FixedCenter}
+        size={Modal.Size.Small}
+        onRequestClose={handleClose}
+      >
+        <Modal.Body>
```

Comment:
I think you should keep `Modal.Body` in a separate component here and use the same Modal wrapper. Then, you can either use it as a step or render it conditionally.

Reply:
Done. I think it would be easier to have a separate component since the modal type of the main component is different from this one.